layout_padded/layout_padded.md

```diff
@@ -2191,76 +2203,176 @@ template<class LayoutRightPaddedMapping>
 
 ## Layout specializations of `submdspan_mapping` [mdspan.submdspan.mapping]
 
-> At the top of Section � [mdspan.submdspan.mapping] ("Layout specializations of `submdspan_mapping`"), before paragraph 1, add the following to the end of the synopsis of specializations.
+> Replace Section � [mdspan.submdspan.mapping] ("Layout specializations of `submdspan_mapping`"), with:
 
-```c++
-  template<class Extents, std::size_t padding_stride, class... SliceSpecifiers>
-    constexpr auto submdspan_mapping(
-      const layout_left_padded<padding_stride>::template mapping<Extents>& src, 
-      SliceSpecifiers ... slices) -> /* see below */;
+<b>24.7.3.7.6 Specialization of `submdspan_mapping` [mdspan.submdspan.mapping]</b>
+
+<b>24.7.3.7.6.1 Common [mdspan.submdspan.mapping.common]</b>
+
+[1]{.pnum} The following elements apply to all functions in [mdspan.submdspan.impl].
```

Comment:
`[mdspan.submdspan.impl]` doesn't exist, either here or in the Standard now.  Did you perhaps mean to write `[mdspan.submdspan.mapping]`?

```suggestion
[1]{.pnum} The following elements apply to all functions in [mdspan.submdspan.mapping].
```